frappe/core/doctype/user/user.py

```diff
@@ -524,8 +524,15 @@ def get_perm_info(role):
 	from frappe.permissions import get_all_perms
 	return get_all_perms(role)
 
+def to_bool(value):
```

Comment:
Is this really needed?

Reply:
I am doing this because logout_all_sessions is of <type 'unicode'> when I am accessing on server side.